requirements.txt

```diff
@@ -1,4 +1,5 @@
 # note there are also rpm packaged deps that aren't
 # setup in pypi or with setup.py
 ../python-rhsm
+decorator
```

Comment:
I've tried to keep requirements.txt up-to-date vis-a-vis the spec file.  Some of the things in the spec file just can't appear in requirements.txt though: e.g. dmidecode or python-ethtool.